rfc6962-bis.xml

```diff
@@ -510,186 +596,170 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
         opaque sct_extension_data&lt;0..2^16-1&gt;;
     } SctExtension;
 
-    SctExtension SctExtensions&lt;0..2^16-1&gt;;
-
     struct {
-        Version sct_version;
         LogID log_id;
         uint64 timestamp;
-        SctExtensions extensions;
+        SctExtension sct_extensions&lt;0..2^16-1&gt;;
         digitally-signed struct {
-            Version sct_version;
-            SignatureType signature_type = certificate_timestamp;
-            uint64 timestamp;
-            LogEntryType entry_type;
-            select(entry_type) {
-                case x509_entry: CertInfo;
-                case precert_entry_V2: CertInfo;
-            } signed_entry;
-            SctExtensions extensions;
-        };
-    } SignedCertificateTimestamp;</artwork>
+            TransItem timestamped_entry;
```

Comment:
General comment: Seems like we're going to great length to accommodate V1 structures by introducing the TransItem -> TransItemV2 abstraction.
Why do you think it's necessary?